api/v1beta1/placementapi_webhook.go

```diff
@@ -121,18 +126,27 @@ func (r *PlacementAPI) ValidateDelete() (admission.Warnings, error) {
 }
 
 func (r PlacementAPISpec) ValidateCreate(basePath *field.Path) field.ErrorList {
-	return r.ValidateDefaultConfigOverwrite(basePath)
+	return ValidateDefaultConfigOverwrite(basePath, r.DefaultConfigOverwrite)
 }
 
 func (r PlacementAPISpec) ValidateUpdate(old PlacementAPISpec, basePath *field.Path) field.ErrorList {
-	return r.ValidateDefaultConfigOverwrite(basePath)
+	return ValidateDefaultConfigOverwrite(basePath, r.DefaultConfigOverwrite)
+}
+
+func (r PlacementAPISpecCore) ValidateCreate(basePath *field.Path) field.ErrorList {
```

Comment:
note to myself: this will be called by the webhook in openstack-operator when validating the service core struct within the openstackcontrolplane struct.